### ISSUE TITLE
Save settings

### ISSUE THREAD

**Issue by Esshahn** [nurpax/petmate#121]
![screenshot 2018-09-14 um 12 45 21](https://user-images.githubusercontent.com/434355/45570761-2a41dd00-b864-11e8-8c05-3e2db178ec90.jpg)

With every restart of the app I need to set the export options again.
If Petmate sports saved settings at some point, these would be a good idea to include.

**Comment by ftoledo** [comment]
I agree.. Save this settings as "default export settings" .. and you can add a check for "prompt on every export y/n"
